pytorch/include_list.pl

```diff
@@ -0,0 +1,65 @@
+#!/bin/perl
+
+# Must be run at from javacpp-presets/pytorch after cppbuild.sh has been run 
+# for linux-x86_64-gpu
```

Comment:
Add it to the cppbuild.sh script

Reply:
This is not fully automatized, there are exceptions to comment out (not parseable headers...)  and it can only run on linux since we need g++.

Reply:
If it's not fully automated, then it means you are manually creating the inlude files. So please fix the include files to preserve the existing order. Either that, or make it automated!

Reply:
Let's make it fully automated then. It will require less work when  a new pytoch version is released and we won't miss new API. 
I'll hard-code the list of exceptions related to non-parseable headers in the script, and move the lists to `src/gen/resources`

Reply:
Look, you're doing all this very ad hocly, which is fine, but like I keep telling, this is equivalent to doing everything manually, so please revert any unnecessary changes! If something doesn't need to be changed, then don't change it. If doesn't matter if you're half-automated tool does half the work, because you're modifying the output of it, so modify it more so the order doesn't change. If you're able to come up with a fully automated method, that's different story. But you're not able to do it fully automated, so until you're able to do that, please revert all and every change that is not necessary. You're editing these files manually already, what's the problem?

Reply:
BTW, why do you say this needs GCC? Is this version of Clang not working for some reason?
http://bytedeco.org/javacpp-presets/llvm/apidocs/org/bytedeco/llvm/program/clang.html

Reply:
> what's the problem?

The problem is that I just spent 1h30 swapping files into an unnatural order, possible introducing bugs, for no good reason. And that I'll have to redo the same thing each new release.
And no, I don't feel like looking to redo the same script using clang, while pytorch is built using g++ and the include list extraction works with g++. When I said I need g++, I meant that the script won't work on Windows.